CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ curl -sL https://github.com/SigNoz/signoz/raw/main/sample-apps/hotrod/hotrod-ins
 ```bash
 kubectl -n sample-application run strzal --image=djbingham/curl \
   --restart='OnFailure' -i --tty --rm --command -- curl -X POST -F \
-  'locust_count=6' -F 'hatch_rate=2' http://locust-master:8089/swarm
+  'locust_count=6' -F 'user_count=2' -F 'spawn_rate=1' -F 'hatch_rate=2' http://locust-master:8089/swarm
```

Comment:
`locust_count` and `hatch_rate` are no longer needed.

```bash
kubectl --namespace sample-application run strzal --image=djbingham/curl \
  --restart='OnFailure' -i --tty --rm --command -- curl -X POST -F \
  'user_count=6' -F 'spawn_rate=2' http://locust-master:8089/swarm
```

Ref: https://signoz.io/docs/install/kubernetes/aws/#optional-install-a-sample-application-and-generate-tracing-data